test/optimize.test.js

```diff
@@ -14,14 +15,5 @@ describe('optimize()', () => {
 
 function optimize(source, options) {
   const svgo = new SVGO(svgOptions(options));
-
-  return new Promise((resolve, reject) => {
-    svgo.optimize(source, (result) => {
-      if (result.error) {
-        return reject(new Error(result.error));
-      } else {
-        return resolve(result.data);
-      }
-    });
-  });
+  return svgo.optimize(source).then((result) => result.data);
```

Comment:
The problem was SVGO.optimize now return [a promise](https://github.com/svg/svgo/blob/master/CHANGELOG.md#---100--30102017)